tests/client-dbus/tests/dbus/pool/test_create_filesystem.py

```diff
@@ -87,8 +88,8 @@ def testDuplicateSpecs(self):
         self.assertEqual(len(result), 1)
         self.assertEqual(rc, StratisdErrors.OK)
 
-        (_, fs_name) = result[0]
-        self.assertEqual(fs_name, new_name)
+        (_, self._FSNAME) = result[0]
```

Comment:
pylint will always be disconcerted to find an assignment to a class-level variable within a test method. Remember that we generally only use identifiers in all caps for constants, and if you are assigning to such an identifier within a method, you're definitely not treating it as a constant. If you look at the scope and the user of the identifer, it is completely local to the method, so you should change back here to the way it was before, and similarly in the method below.